### PR TITLE
Setup "Stale" probot

### PR DESCRIPTION
**Pull request by pamil**
| Q               | A
| --------------- | -----
| Branch?         | 1.0
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | -
| License         | MIT

This one will help to focus on the most important issues and pull requests.

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+daysUntilStale: 90
+daysUntilClose: 7
+exemptLabels:
+  - Critical
+  - RFC
```
Comment (lchrusciel) [inline .github/stale.yml]: Shouldn't we also leave the `Feature` label?

Comment (pamil) [reply .github/stale.yml]: This probot works in two phases:
 - when an issue / PR is not updated for 90 days, it posts a comment and marks it as stale
 - when a stale issue / PR is not updated for another 7 days, it is closed
 - when a stale issue / PR is updated within 7 days, it removes stale label and the whole process starts again

So the reasoning behind it was to close those issues / PRs that people are not longer interested in. If there's still an interest, only four actions in a year would prevent it from closing :)

Comment (CoderMaggie) [reply .github/stale.yml]: :+1:

**Comment by pamil** [comment]
For anyone disturbed by tons of new notifications, make use of this JS script which removes the notifications when last reply is from Stalebot:

```
[...document.querySelectorAll('.list-group-item.js-notification.js-navigation-item.unread')].filter(_ => _.innerHTML.includes('Sylius/Sylius')).filter(_ => [..._.querySelectorAll('.avatar')].shift().href.match(/\/stale\[bot\]$/)).forEach(_ => _.querySelector('.delete-note').click())
```

**Pull request merged**